Assets/PatchKitTools/Editor/Views/SelectApp.cs

```diff
@@ -57,57 +58,67 @@ private void Reload()
                     .ToList();
         }
 
+        //private bool FindDuplicateApp(string name)
+        //{
+        //    var api = _api.GetApps();
+        //    for (int i = 0; i < api.Count; i++)
+        //    {
+        //        if (api[i].Name == name) return true;
+        //    }
+        //    return false;
+        //}
         public void Show()
         {
-            EditorGUILayout.LabelField("New app: ", EditorStyles.boldLabel);
-
-            newAppName = EditorGUILayout.TextField("Name: ", newAppName);
-
-            if (string.IsNullOrEmpty(newAppName))
-            {
-                EditorGUILayout.HelpBox("Application name cannot be empty.", MessageType.Error);
-            }
-            else
+            
+            if (GUILayout.Button(new GUIContent("←", "Change application"), GUILayout.Width(40)))
             {
-                if (GUILayout.Button("Add"))
-                {
-                    var newApp = _api.CreateNewApp(newAppName, EditorUserBuildSettings.activeBuildTarget.ToPatchKitString());
-                    if (OnAppSelected != null) OnAppSelected(newApp);
-                }
+                if (OnChangeApp != null) OnChangeApp();
             }
-            
-            EditorGUILayout.Separator();
-            
-            GUILayout.Label("Your apps: ", EditorStyles.boldLabel);
+            EditorGUILayout.BeginHorizontal();
+            GUILayout.FlexibleSpace();
+                GUILayout.Label("\n Choose target PatchKit application from list below. \n", EditorStyles.boldLabel);
```

Comment:
Why is this line indented, but `GUILayout.FlexibleSpace();` above and below is not?